src/logic/function/conjugate.lean

```diff
@@ -51,7 +51,8 @@ lemma inverses_right (h : semiconj f ga gb) (ha : right_inverse ga' ga)
 
 end semiconj
 
-/-- Two maps `f g : α → α` commute if `f ∘ g = g ∘ f`. -/
+/-- Two maps `f g : α → α` commute if `f ∘ g = g ∘ f`.
+Given `h : function.commute f g` and `a : α`, the term `h a` has Type `h a : f (g a) = g (f a)`. -/
```

Comment:
```suggestion
Given `h : function.commute f g` and `a : α`, the term `h a` has Type `f (g a) = g (f a)`. -/
```
or perhaps
```suggestion
Given `h : function.commute f g` and `a : α`, we have `h a : f (g a) = g (f a)`. -/
```
otherwise this reads a bit like `h a : h a` which is nonsense!